frontend/src/constants/workflows.ts

```diff
@@ -0,0 +1,7 @@
+/** Property type enum */
+export enum Workflows {
```

Comment:
Might need to include the `ASSESS-EX-DISPOSAL` workflow too

Reply:
ok, presumably that should trigger the ERP red map markers?

Reply:
Ill just include this in my next PR.